File: server-mh/src/Match/Handler/Models/DiceState.ts

export class DiceState {
    public waitingForInput: boolean;
    public waitingForAnimation:boolean;
    public diceValue: number;
    public waitingForActionSelect: boolean;

    constructor(
    ) {
        this.waitingForInput = false;
        this.waitingForAnimation=false;
        this.diceValue = 0;
        this.waitingForActionSelect = false;
    }
}

File: Client-unity/Ludo/Assets/Scripts/OfflineMatchHandler/DiceState.cs
namespace Ludo.Offline
{
    public class DiceState
    {
        public bool WaitingForInput;
        public bool WaitingForAnimation;
        public int DiceValue;
        public bool WaitingForActionSelect;

        public DiceState()
        {
            WaitingForInput = false;
            WaitingForAnimation = false;
            DiceValue = 0;
            WaitingForActionSelect = false;
        }
    }
}